app/src/main/java/org/stepik/android/presentation/course_list/CourseListPresenter.kt

```diff
@@ -67,6 +78,10 @@ constructor(
         val oldState = state as? CourseListView.State.Content
             ?: return
 
+        if (oldState.courseListItems.any { it is CourseListItem.PlaceHolder }) {
```

Comment:
it seems more effective to check only last item for `CourseListItem.PlaceHolder`